integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/metamodel/IndexFieldDescriptorIT.java

```diff
@@ -98,7 +98,8 @@ public void valueField() {
 		assertThat( fieldDescriptor )
 				.returns( getRelativeFieldName(), IndexFieldDescriptor::relativeName )
 				.returns( getAbsoluteFieldPath(), IndexFieldDescriptor::absolutePath )
-				.returns( fieldStructure.isMultiValued(), IndexFieldDescriptor::multiValued );
+				.returns( fieldStructure.isMultiValued(), IndexFieldDescriptor::multiValued )
+				.returns( fieldStructure.isMultiValuedInRoot(), IndexFieldDescriptor::multiValuedInRoot );
```

Comment:
Maybe you could test this for object fields too (i.e. in the `parent()` test method)?